packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -284,9 +284,16 @@ const AccountWizard = props => {
                     props.onFinishSuccess(response.new_account_real.currency.toLowerCase());
                 }
                 const { document_type, document_number, document_additional } = { ...form_values() };
-                if (document_type && document_type.id !== IDV_NOT_APPLICABLE_OPTION.id && document_number) {
+                /**
+                 * If user opted-out of IDV verification, we send the value "none" for document_number and document_type to the API.
+                 */
+                const idv_doc_number =
+                    document_type.id === IDV_NOT_APPLICABLE_OPTION.id
+                        ? IDV_NOT_APPLICABLE_OPTION.value
+                        : document_number;
+                if (document_type && idv_doc_number) {
```

Comment:
can we do this formating in idv component itself and not in createRealAccount function?

Reply:
Valid suggestion. Will check on it